bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -151,13 +162,21 @@
 		<label>Heating Power</label>
 		<description>Current heating power</description>
 		<category>Fire</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
```

Comment:
Is it that really? I would expect measurement of power only for `Number:Power`. Is it a percentage?

Reply:
Yes it is a percentage of the maximum heating output power of the boiler.

Reply:
@jlaur how shall we proceed?

Reply:
I can't recommend better tags, but measurement of power seems wrong, so I'd propose to remove the tags.